plugin.rb

```diff
@@ -234,7 +235,7 @@ def include_event?
     end
 
     def event_going
-      object.topic.event_going
+      User.find(object.topic.event_going).pluck(:username) if object.topic.event_going
```

Comment:
@fzngagan Why are we serializing a seperate event_going here instead of using the ``going`` property of the event object we're already serializing? I understand that this was part of the existing structure, but considering we're changing this, and adding a query directly into the serializer itself (which you typically want to avoid), the quetion arises as to what purpose it is serving?